_includes/disclaimer.html

```diff
@@ -1,5 +1,12 @@
-<div class="disclaimer">
+<header class="site-header">
```

Comment:
Agreed. But since the hierarchy on this is really `<body>` it might be nice to distinguish it specifically from other headers. This is similar to the way that openFEC did implemented it: https://github.com/18F/openFEC-web-app/blob/develop/templates/layouts/main.html#L63

Happy to move forward without it, but wanting to try to think ahead for future codings.